modules/assignment/src/main/resources/ehcache.xml

```diff
@@ -19,7 +19,9 @@
          xsi:noNamespaceSchemaLocation="http://ehcache.org/ehcache.xsd"
          updateCheck="true"
          monitoring="autodetect"
-         dynamicConfig="true">
+         dynamicConfig="true"
+         maxBytesLocalHeap="2G"
```

Comment:
2gb is just the best guess higher limit.. We currently run app instances with 8gb of memory, out of which consistently 1.1gb is being used...